Models/fetchArticleId.models.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+
+function fetchArticleId(article_id) {
+  return db
+    .query(
+      `SELECT * FROM articles
+    WHERE article_id = $1`,
+      [article_id]
+    )
+    .then((result) => {
+      if (result.rowCount < 1) {
```

Comment:
Nice work! You could be more explicit and check that` result.rowCount === 0` no need to change this, just a suggestion!